GreenhouseNode/GreenhouseNode/handlers/HumidityHandler.c

```diff
@@ -84,21 +84,22 @@ humidityHandler_t humidityHandler_create(UBaseType_t hum_task_priority, EventGro
 	return newReader;
 }
 
+uint16_t humidityHandler_getHumidity(humidityHandler_t self, humidityHandler_t humToDestroy)
+{
+	humToDestroy->humidity;
+	return self -> humidity;
+}
+
 humidityHandler_t humidityHandler_destroy(humidityHandler_t self)
 {
 	if(self == NULL)
 	{
 		return;
-		vTaskDelete(self ->tempToDestroy);
+		vTaskDelete(self ->humToDestroy);
 		vPortFree(self);
```

Comment:
Is this event a FreeRTOS task? Because I believe these methods can only be used inside those. I think what you meant to do it `free(self)`.